openmrs-watcher/src/test/java/org/openmrs/eip/mysql/watcher/DebeziumMessageProcessorTest.java

```diff
@@ -157,7 +160,8 @@ public void process_shouldCreateAnEventAndAddItAsAHeaderForAnInsert() throws Exc
 		Event event = exchange.getProperty(PROP_EVENT, Event.class);
 		assertEquals(table, event.getTableName());
 		assertEquals(id.toString(), event.getPrimaryKeyId());
-		assertEquals(uuid, event.getIdentifier());
+		// IdentifierSettingProcessor should have set the identifier to the uuid
+		// assertEquals(uuid, event.getIdentifier());
```

Comment:
Is there a better way to assert this than comment it out?

Reply:
`assertNull(event.getIdentifier());` Since it's the responsibility of `IdentifierSettingProcessor` to set the Identifier. At this point, should be always null.